src/react-app-env.d.ts

```diff
@@ -1 +1,22 @@
-/// <reference types="react-scripts" />
+interface Coment {
```

Comment:
do not allow grammatic mistakes, just name it differently

```suggestion
interface CommentItem {
```